WordPress/wordpress-playground
# Blueprints: Remove setPhpIniEntry step

Removes the `setPhpIniEntry` step. It doesn't work reliably – setting PHP.ini entries must happen before the PHP is initialized, but Blueprints are executed after PHP is initialized. The existing Blueprints will continue to work – the Blueprints compiler will ignore the `setPhpIniEntry` step and issue a console warning to notify the developer.

To set php.ini entries, either customize the php.ini files or use an mu-plugin.

 ## Testing instructions

Run a Blueprint with a setPhpIniEntry step, confirm the Playground still runs the rest of the Blueprint.
